### ISSUE TITLE
OS X 12 Monterey support?

### ISSUE THREAD

**Issue by jli** [briankendall/forceFullDesktopBar#15]
Just wondering if this supports Monterey? I couldn't find docs or discussion on this. Thanks!

**Comment by AJAYK-01** [comment]
hey, would this work if i enable sip after the installation and code injection?

**Comment by briankendall** [comment]
No, the code injection occurs every time the Dock launches, so SIP has to be disabled every time your system boots up.
